components/card-faskes.js

```diff
@@ -0,0 +1,53 @@
+export default function CardFaskes() {
+  return (
+    <div id="card-faskes" className="px-8 py-4">
+      <h3 className="text-2xl font-bold">Data Ketersediaan Fasilitas Kesehatan</h3>
```

Comment:
Delete any section header.. This PR should working on Card Componets for Faskes data